modules/ROOT/pages/security/proc_self-signed-certs.adoc

```diff
@@ -0,0 +1,79 @@
+[[using-self-signed-certificates-in-mobile-clients]]
+== Using Self-Signed Certificates in Mobile Clients
+
+Throughout the development lifecycle of a mobile applications, a common
+requirement is often the need to integrate and connect with some backend
+services, often in a secure manner. However in order for the client
+device to connect over SSL/TLS, it will need to trust the root
+certificate authority. This cert must be installed on the client device
+to ensure that the client trusts server certificates that are signed by
+the CA.
+
+Normally, this is not an issue and is done automatically. In a
+development environment, such as a local OpenShift cluster which uses
+self-signed certificates, this is not the case. However we have the
+ability to easily extract the root certificate from the cluster allowing
+us to install it on the device and mark it as trusted, allowing our
+application and/or web browser to connect over a secure layer.
+
+[[extracting-root-ca]]
+== Extracting Root CA
+The simplest method for extracting the required root certificate is to
+query the API server for the OpenShift routers secret and parse the
+result. The following command, which parses the response and saves it to
+a file `localcluster.crt` in the `/tmp` directory should be run as
+`system:admin` user.
+
+[source,bash]
+----
+oc get secret router-certs --template='{{index .data "tls.crt"}}' -n default  |  \
```

Comment:
IMHO - this should be some script in mobile core. It works now but i could outdate really fast.